wikipedia/challenges/default.json

```diff
@@ -16,25 +16,25 @@
       "operation": "check-cluster-health"
     },
     {
-      "name": "index-documents",
-      "operation": "index-documents",
-      "warmup-time-period": {{bulk_warmup | default(40) | int}},
-      "clients": {{bulk_indexing_clients | default(5)}}
+      "name": "initial-documents-indexing",
+      "operation": "initial-documents-indexing",
+      "warmup-time-period": {{initial_indexing_bulk_warmup | default(40) | int}},
+      "clients": {{initial_indexing_bulk_indexing_clients | default(5) | int}}
     },
     {
       "name": "refresh-after-index",
       "operation": "refresh-after-index"
     },
     {
-      "name": "query-string-search",
-      "operation": "query-string-search",
-      "clients": {{search_clients | default(20)}},
-      "time-period": {{search_time_period | default(300) | int}},
-      "warmup-time-period": {{search_warmup | default(10) | int}}
+      "name": "wait-until-merges-finish-after-index",
```

Comment:
ℹ️ Waiting for merge operations to be finished after the initial indexing. 

We are not doing an explicit force merge but want to wait for automatic merges to be finished so it will not bias the results.